labs/pluggable-hcs

```diff
@@ -0,0 +1,23 @@
+# Lab Name
+_Pluggable Hedera Consensus Service_
+
+# Short Description
+_The Pluggable HCS Lab proves the ability of a Hyperledger Fabric network to connect to the Hedera Consensus Serivce running on a publicly accessilbe testnetwork._
+
+# Scope of Lab
+_This lab is designed to enable a pluggable consensus option within Hyperledger Fabric. The Hedera Consensus Service is available on the Hedera public network. The Lab initially uses the Build Your First Network (BYFN) Hyperledger Fabric sample to connect a set of Orderers to the Hedera Consensus Service. The demo proves the ability of HCS to deliver a set of ordered transactions from the public network to a Fabric network for block creation and propogation._
+
+# Initial Committers
+_Enter the Github IDs for the set of initial committers._
+- https://github.com/xin-hedera
+- https://github.com/mikeburragejr
+- https://github.com/donaldthibeau
+
+# Sponsor
+_Provide the name of your sponsor. A sponsor is required and must be a maintainer of one of the Hyperledger projects, a TSC member, or a WG chair._
+- https://github.com/christo4ferris - Hyperledger Technical Steering Committee Member
+
+# Pre-existing repository
+_If you currently have a Github repository that you wish to transfer to the Hyperledger Labs organization, please provide a link here. **NOTE: Please refer to the README for additional information on existing repositories.**_
+- https://github.com/hashgraph/fabric-samples-hcs/tree/master/first-network
```

Comment:
Are you saying you want to move the entire https://github.com/hashgraph/fabric-samples-hcs repository to Hyperledger Labs? Secondly, can you confirm that this repository has DCO signoff on all commits?

Reply:
Yes, we need the entire repo. In addition, we need the following repo as well:
https://github.com/hashgraph/fabric-hcs

The commits don't have DCO signoff. I'll start working on that.

Reply:
all the commits in fabric-samples-hcs and fabric-hcs have my DCO signoff now.